src/shared/src/components/LandmarkShortcuts.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import type { ILandmarkShortcutsProps } from "./LandmarkShortcuts";
+import { LandmarkShortcuts } from "./LandmarkShortcuts";
+
+describe('LandmarkShortcuts.tsx', () => {
+  it('should render supplied text', () => {
+    renderMainContentNav();
+    expect(screen.getByText('Hopp til hovedinnhold')).toBeInTheDocument();
+  });
+
+  it('should set active element to main when nav is clicked', async () => {
+    renderMainContentNav();
+    await userEvent.click(screen.getByRole('link'));
+    expect(document.activeElement).toBe(document.getElementById('main-content'));
+  });
+
+  function renderMainContentNav(props: Partial<ILandmarkShortcutsProps> = {}) {
+    const defaultProps: ILandmarkShortcutsProps = {
+      navToMainText: 'Hopp til hovedinnhold'
```

Comment:
I think this landmark api could be easier to config:
this could be an array with `[ {id, shortcutText} ]`.
```js
[{ id: 'main-content', shortcutText: 'Hopp til hovedinnhold'}]
```
Then it would be easier to extend to other landmarks later.
Some changes would have to be made in LandmarkShortcuts to iterate through these.

Reply:
Done ✔️